ui-v2/src/components/flows/cells.tsx

```diff
@@ -75,13 +80,80 @@ export const FlowDeploymentCount = ({ row }: { row: { original: Flow } }) => {
 	return data;
 };
 
+const DeleteMenuItem = ({
+	id,
+	name,
+	onClose,
+}: {
+	id: string;
+	name: string;
+	onClose?: () => void;
```

Comment:
This needs to be passed in so that we can close the menu when "close" or "delete" is clicked. Has to do with this issue https://github.com/radix-ui/primitives/issues/1836